### DISCUSSION TITLE
modcharts

### DISCUSSION THREAD

**Discussion by swobodac** [DEMOLITIONDON96/Demolition-Engine#123] [Lua Q&A] [answered]
lua script for opposition modchart:https://www.youtube.com/watch?v=ipc0LA2-ybY&ab_channel=Pianoofied

**Comment by REALHaydenGaming** [comment, marked as answer]
you are the most fucking greediest person for stealing code. NO.

**Reply by ThatOneSillyGuyDon** [reply]
gg.
